### PR TITLE
[CMS-706] Update installer to use phar file instead of composer.

### PR DESCRIPTION
**Pull request by kporras07**

### CODE REVIEW

**Review by namespacebrian — APPROVED** [review]
I assume it works :)

File: src/Utils/TerminusPackage.php [hunk 1]
```diff
@@ -51,20 +49,12 @@ public function getInstallDir()
...
-    /**
-     * @return mixed Returns the version of Terminus currently installed
-     */
-    public function getInstalledVersion()
-    {
-        return str_replace('* ', '', $this->getOutdatedInfo()['versions']);
-    }
-
     /**
      * @return string Latest version of Terminus according to Composer
```
Comment (greg-1-anderson) [inline src/Utils/TerminusPackage.php:53]: Change the comment -- this method is now responsible for returning the initial version of Terminus to download from GitHub. After that, self:update is used to. get to the latest version.

Comment (greg-1-anderson) [reply src/Utils/TerminusPackage.php:53]: Could also rename it to `getInitialDownloadVersion()` or something like that.

**Review by greg-1-anderson — APPROVED** [review]
LGTM, just ensure that comments are updated to match code changes.

**Pull request merged**